internal/generator/forwarder/generator.go

```diff
@@ -25,6 +25,7 @@ var (
 	ErrTLSOutputNoSecret = func(o logging.OutputSpec) error {
 		return fmt.Errorf("No secret defined in output %s, but URL has TLS Scheme %s", o.Name, o.URL)
 	}
+	ErrGCL = errors.New("Exactly one of billingAccountId, folderId, organizationId, or projectId must be set.")
```

Comment:
We must set one in the set or we can choose to set none or one?

Reply:
Exactly one from the set.